pkg/trace/agent/run.go

```diff
@@ -94,22 +94,22 @@ func Run(ctx context.Context) {
 	rand.Seed(time.Now().UTC().UnixNano())
 
 	agnt := NewAgent(ctx, cfg)
-	log.Infof("trace-agent running on host %s", cfg.Hostname)
+	log.Infof("Trace-agent running on host %s", cfg.Hostname)
```

Comment:
I think this case should stay lower cased. The binary name is `trace-agent`.

Reply:
Maybe it should be change to "Trace agent" then 🤔

Reply:
I think I'd be fine with either. If we use the former, the most correct would be to simply use `os.Args[0]` which is the binary name. I'm ok with either. Up to you. 🙇